.fantomasignore

```diff
@@ -0,0 +1,2 @@
+test/
+obj/**/*.fs
```

Comment:
Instead of specifying specific files, let's omit files so we always ignore these no matter if it's from the tool or in the editor.